Hearthstone Deck Tracker/BobsBuddy/BobsBuddyInvoker.cs

```diff
@@ -187,6 +195,9 @@ public async void StartCombat()
 						DebugLog($"Found hero power trigger for {minion.minionName} after {duration}ms");
 				}
 
+				if(_game.Opponent.Board.FirstOrDefault(x => x.IsHeroPower)?.CardId.Contains(LichKingHeroPowerId) ?? false)
+					await Task.Delay(2000);
+				_currentOpponentMinions = null;
```

Comment:
Is there any reason for this to ever be null? I think you can just initialize it at the top and not worry about it. Since this instance is scoped to the current turn there would only ever be one opponent.